src/clj/rems/application/events.clj

```diff
@@ -2,54 +2,59 @@
   (:require [clojure.test :refer :all]
             [schema-refined.core :as r]
             [schema.core :as s]
+            [rems.api.schema :refer [EventBase]]
```

Comment:
I realise the need for code sharing but I'm not sure I like rems.application depending on rems.api. Maybe rems.schema would be a good place for "fundamental" schema definitions?

The reason I think there should be no dependency is that we want to keep the API stable, but we are free to change our internal representations as long as we produce the same API.

Reply:
Done.